style.css

```diff
@@ -83,6 +84,56 @@ main {
     }
   }
 
+  #menuChild {
+    visibility: hidden;
+    width: 100%;
+    background-color: #6070ff;
+    height: 100vh;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    display: flex;
+    flex-direction: column;
+    color: white;
+    padding-top: 24%;
+    opacity: 0.7;
+  }
+
+  li.menuItem1,
+  li.menuItem2,
+  li.menuItem3 {
+    list-style-type: none;
+    cursor: pointer;
+    margin-right: 4vh;
+    font-weight: 500;
+    font-size: 15px;
+    transition: 0.3s;
+    margin-left: 32px;
+  }
+
+  li.menuItem2,
+  li.menuItem3 {
+    margin-top: 20px;
+  }
+
+  #cancel-icon {
+    position: absolute !important;
+    top: 10%;
+    right: 10%;
+    z-index: 20;
+  }
+
+  .a {
+    text-decoration: none;
+    font-size: 32px;
+    color: #fff;
+  }
+
+  .blur {
+    filter: blur(5px);
+  }
+
```

Comment:
Good job adding mobile menu to your design so far! There are however, few thing we need to improve upon with regards to the mobile menu.
- [ ] When a mobile menu item is selected the jumped to section still stays blurred. Let's ensure to remove the overlay and blurred feature once a mobile menu is clicked
- [ ] Let's also ensure that the _hamburger_ and the _close_ button are in the same position when toggled, so as to enhance the UX. 

**After menu item is clicked:**
**Portolio section:**        |  **About section:**        |  **Contact section:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/223386292-1024f92c-34c5-4828-878d-12dd15a60b68.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/223384265-36dfb219-bc54-4977-845a-f28889d7bc3f.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/223386563-8030e6b3-434a-4091-97fb-cbbf533ba6c2.png">


**Hamburger and close (X) button not aligned:**
**Portolio section:**        |  **About section:**        
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/223385433-af7c5e23-94df-4a91-bb6d-cf35306efbcb.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/223385817-09f7b8ea-627c-4bcb-a247-0982ced0d228.png">